### ISSUE TITLE
Apply namespacing to Box2D

### ISSUE THREAD

**Issue by Bambo-Borris** [Bambo-Borris/box2d#7]
Currently Box2D has no namespace and uses a naming convention of `b2ClassName`. Preferred alternative would be to forgo the `b2` prefix, and apply a namespace to all Box2D types.

**Comment by ChrisThrasher** [comment]
Just `namespace b2` so we can remove the `b2_` prefix from all identifiers.

**Comment by Bambo-Borris** [comment]
Yeah exactly what @ChrisThrasher was referring to is how I envision the change
